src/garrysmod/gamemodes/zombiesurvival/gamemode/cl_scoreboard.lua

```diff
@@ -1,216 +1,249 @@
-	local curpan = self:GetPlayerPanel(pl)
-	if curpan and curpan:IsValid() then return curpan end
+PANEL = {}
 
-	if pl:Team() == TEAM_SPECTATOR then return end
+function PANEL:Init()
+    local screenscale = math.max(0.95, BetterScreenScale())
 
-	local panel = vgui.Create("ZSPlayerPanel", pl:Team() == TEAM_UNDEAD and self.ZombieList or self.HumanList)
-	panel:SetPlayer(pl)
-	panel:Dock(TOP)
-	panel:DockMargin(8, 2, 8, 2)
+    self.m_SpecialImageContainer = vgui.Create("Panel", self)
+    self.m_SpecialImageContainer:Dock(LEFT)
+    self.m_SpecialImageContainer:SetWide(20)
 
-	self.PlayerPanels[pl] = panel
+    self.m_AvatarButtonContainer = vgui.Create("Panel", self)
+    self.m_AvatarButtonContainer:Dock(LEFT)
+    self.m_AvatarButtonContainer:SetWide(32 * screenscale)
 
-	return panel
-end
+    self.m_PlayerLabelContainer = vgui.Create("Panel", self)
+    self.m_PlayerLabelContainer:Dock(FILL)
+    self.m_PlayerLabelContainer:DockMargin(4, 0, 0, 0)
 
-function PANEL:RefreshScoreboard()
-	self.m_ServerNameLabel:SetText(GetHostName())
-	self.m_ServerNameLabel:SizeToContents()
-	self.m_ServerNameLabel:SetPos(math.min(self:GetWide() - self.m_ServerNameLabel:GetWide(), self:GetWide() * 0.75 - self.m_ServerNameLabel:GetWide() * 0.5), 32 - self.m_ServerNameLabel:GetTall() / 2)
+    self.m_PingMeterContainer = vgui.Create("Panel", self)
+    self.m_PingMeterContainer:Dock(RIGHT)
+    self.m_PingMeterContainer:SetWide((32 * screenscale) - 4)
+
+    self.m_MuteContainer = vgui.Create("Panel", self)
+    self.m_MuteContainer:SetWide(24)
+    self.m_MuteContainer:Dock(RIGHT)
 
-	if self.PlayerPanels == nil then self.PlayerPanels = {} end
+    self.m_FriendContainer = vgui.Create("Panel", self)
+    self.m_FriendContainer:Dock(RIGHT)
+    self.m_FriendContainer:SetWide(24)
 
-	for pl, panel in pairs(self.PlayerPanels) do
-		if not panel:IsValid() or pl:IsValid() and pl:IsSpectator() then
-			self:RemovePlayerPanel(panel)
-		end
-	end
+    self.m_ClassImageContainer = vgui.Create("Panel", self)
+    self.m_ClassImageContainer:Dock(RIGHT)
+    self.m_ClassImageContainer:SetWide(22 * screenscale)
 
-	for _, pl in pairs(player.GetAllActive()) do
-		self:CreatePlayerPanel(pl)
-	end
+    self.m_RemortLabel = EasyLabel(self, "Remort", "ZSScoreBoardPlayer", COLOR_WHITE)
+    self.m_RemortLabel:Dock(RIGHT)
+    self.m_RemortLabel:SetWide(50)
+    self.m_RemortLabel:SetContentAlignment(4)
+
+    self.m_ScoreLabel = EasyLabel(self, "Score", "ZSScoreBoardPlayer", COLOR_WHITE)
+    self.m_ScoreLabel:Dock(RIGHT)
+    self.m_ScoreLabel:SetWide(50)
+    self.m_ScoreLabel:SetContentAlignment(4)
 end
 
-function PANEL:RemovePlayerPanel(panel)
-	if panel:IsValid() then
-		self.PlayerPanels[panel:GetPlayer()] = nil
-		panel:Remove()
-	end
+function PANEL:SetTeam(team)
+    if team == TEAM_UNDEAD then
+        self.m_ScoreLabel:SetText("Brains")
+    end
 end
 
-vgui.Register("ZSScoreBoard", PANEL, "Panel")
+vgui.Register("ZSPlayerPanelHeader", PANEL, "Panel")
```

Comment:
This is a new VGUI element that has the headers for the columns. I basically took all the elements from a player card, removed any real content, so I could just keep the size of the containers.